packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -769,3 +785,8 @@ export class MetriportMedicalApi {
     return new WebhookRequestParsingFailure(parse.error, parse.error.format());
   }
 }
+
+function getPaginationParams(pagination?: Pagination) {
```

Comment:
why do you want this? the spread operator works on undefined.

Reply:
To make sure we're only getting the properties we want from that parameter